oura-library/src/main/kotlin/org/radarbase/oura/request/OuraRequestGenerator.kt

```diff
@@ -242,8 +242,8 @@ constructor(
     companion object {
         private val logger = LoggerFactory.getLogger(OuraRequestGenerator::class.java)
         private val BACK_OFF_TIME = Duration.ofMinutes(10L)
-        private val ONE_DAY = 1L
-        private val TIME_AFTER_REQUEST = Duration.ofDays(10)
+        private val ONE_DAY = Duration.ofDays(1L)
```

Comment:
i think you forgot to update the `userNextRequest[request.user.versionedId] = Instant.now().plus(BACK_OFF_TIME)` with `ONE_DAY ` on line 154.